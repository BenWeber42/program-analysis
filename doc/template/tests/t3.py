def f(x, y):
    return x + y + 1, x

def f_inv (p, q) :
    return unknown_choice(p,q) , unknown_choice(p,q) + unknown_choice(p,q,-p,-q) + unknown_int()