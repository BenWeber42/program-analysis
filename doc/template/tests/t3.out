3 1
5 2
7 3
1 0
2 0
3 0
2000 999
2001 1000
